src/Sylius/Bundle/ShopBundle/Resources/views/Order/thankYou.html.twig

```diff
@@ -24,6 +24,13 @@
                 <div class="ui hidden divider"></div>
                 <a href="{{ path('sylius_shop_order_show', {'tokenValue': order.tokenValue}) }}" id="sylius-show-order" class="ui large blue button">{{ 'sylius.ui.view_your_order_or_change_payment_method'|trans }}</a>
             {% endif %}
+
+            {% if order.customer.user is null %}
+                <a href="{{ path('sylius_shop_register_after_checkout', {'customerId': order.customer.id, 'addressId': order.billingAddress.id}) }}" class="ui large green button">
```

Comment:
Given we use autoincremented identifiers for both customer and address, one might use this feature to scrap addresses, which should be prevented. One idea is to save that data in session instead of passing it through GET arguments.